jobs/e2e_node/containerd/containerd-release-1.6-systemd-cgroup-presubmit/env

```diff
@@ -0,0 +1,9 @@
+CONTAINERD_TEST: 'true'
+CONTAINERD_LOG_LEVEL: 'debug'
+CONTAINERD_DEPLOY_PATH: 'k8s-staging-cri-tools'
+CONTAINERD_PKG_PREFIX: 'containerd-cni'
+
+CONTAINERD_EXTRA_RUNTIME_HANDLER: 'test-handler'
+CONTAINERD_EXTRA_RUNTIME_OPTIONS: |
+  BinaryName = "/home/containerd/usr/local/sbin/runc"
+CONTAINERD_SYSTEMD_CGROUP: 'true'
```

Comment:
What about the `CONTAINERD_CGROUPV2` env. Should we use it to setup cgroupv2?

Reply:
COS M117 defaults to cgroupv2 and it doesn't look like we're setting this explicitly in the other jobs either.  But I can add it here.